app/org/sagebionetworks/bridge/play/controllers/ConsentController.java

```diff
@@ -25,36 +26,41 @@
     private ParticipantOptionsService optionsService;
 
     @Autowired
-    public void setConsentService(ConsentService consentService) {
+    final void setConsentService(ConsentService consentService) {
         this.consentService = consentService;
     }
     @Autowired
-    public void setOptionsService(ParticipantOptionsService optionsService) {
+    final void setOptionsService(ParticipantOptionsService optionsService) {
         this.optionsService = optionsService;
     }
 
+    @Deprecated
     public Result getConsentSignature() throws Exception {
         final UserSession session = getAuthenticatedAndConsentedSession();
         final Study study = studyService.getStudy(session.getStudyIdentifier());
-
-        ConsentSignature sig = consentService.getConsentSignature(study, session.getUser());
-        return ok(ConsentSignature.SIGNATURE_WRITER.writeValueAsString(sig));
+        return getConsentSignatureV2(study.getIdentifier());
```

Comment:
getConsentSignatureV2() already calls getAuthenticatedAndConsentedSession() and already calls getStudy(). We can delete those calls from getConsentSignature() to avoid duplicate work.